tests/conftest.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+import logging
+
+from custom_components.hubspace import hubspace
+
+
+@pytest.fixture
+def mocked_hubspace(mocker):

```

Comment:
Since I considered `HubSpace` a boundary, I opted to mock the connection so github secrets would not be required for testing. Ideally this would be pushed into its own repository and would become a hard boundary.